src/core/stdc/stdlib.d

```diff
@@ -159,6 +159,17 @@ else
 ///
 void*   malloc(size_t size) pure;
 ///
+void*   pureMalloc(size_t size) pure
+{
+    const errno = pureGetErrno();
+    typeof(return) ret = malloc(size);
```

Comment:
Why not use `void*` or `auto` here? `malloc`'s return type isn't going to change anytime soon .